File: CursoPOO/php/Car.php
<?php
require_once("account.php");
class Car {
  public $id;
  public $license;
  public $driver;
  protected $passengers;

  public function __construct($license, $driver) {
      $this->license = $license;
      $this->driver = $driver;
    }

  public function PrintDataCar(){
      echo "
          Licencia: $this->license
          Driver: {$this->driver->name}
          Número de pasajeros: $this->passengers
          
      ";
    }

  public function getPassenger() {
    return $this->passengers;
  }

  public function setPassenger($passenger) {
        
    if ($passenger == 4) {
        $this->passengers = $passenger;
    }
    else {
        echo "Necesitas asignar 4 pasajeros";
    }

  }
}
?>

File: CursoPOO/java/Car.java
class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }
    void printDataCar() {
        if (passenger != null){
            System.out.println("License: " + license + "Driver: " 
            + driver.name);
        }
 
    }

    public Integer getPassenger() {
        return passenger;
    }
    public void setPassenger(Integer passenger) {
        if (passenger == 4) {
            this.passenger = passenger;
        }else{
            System.out.println("Debes ingresar 4 pasajeros");

        }


    }
}